todo_app/data/session_items.py

```diff
@@ -1,82 +1,173 @@
-from flask import session
 
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
-
-def get_items():
-    """
-    Fetches all saved items from the session.
-
-    Returns:
-        list: The list of saved items.
-    """
-    return session.get('items', _DEFAULT_ITEMS.copy())
-
-
-def get_item(id):
-    """
-    Fetches the saved item with the specified ID.
-
-    Args:
-        id: The ID of the item.
-
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-
-
-def add_item(title):
-    """
-    Adds a new item with the specified title to the session.
-
-    Args:
-        title: The title of the item.
-
-    Returns:
-        item: The saved item.
-    """
-    items = get_items()
-
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
-
-    # Add the item to the list
-    items.append(item)
-    session['items'] = items
-
-    return item
-
-
-def save_item(item):
-    """
-        Updates an existing item in the session. If no existing item matches the ID of the specified item, nothing is saved.
-
-    Args:
-        item: The item to save.
-    """
-    existing_items = get_items()
-    updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
-
-    session['items'] = updated_items
-
-    return item
-
-# Delete function to delete an item from the list
-
-def delete_item(itemid):
-    
-    items = get_items()
-    for i in range(len(items)):
-        if items[i]['id'] == itemid:
-            del items[i]
-            break
-    items = get_items()
-    session['items'] = items
-    return items        
\ No newline at end of file
+import requests
+import os
+import json
+from dotenv import load_dotenv
+
+#Class to define propoerties of item created in Trello 
+class MyItem:
+   def __init__(self,itemid,title,status):
+        self.id = itemid
+        self.title = title
+        self.status = status
+   
+   def displayitem(self):
+       print (self.id, self.title,self.status)
+  
+#loading environment variables for KEY,TOKEN to be used if functions are invoked without flask
+load_dotenv()
+
+#Get id of board created in Trello
+def getBoardid():
+   url_for_board = 'https://api.trello.com/1/members/me/boards'
+
+
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN')
+   }
+
+   response = requests.request(
+      "GET",
+      url_for_board,
+      params=query
+   )
+   
+   getBoardresponse = response.text
+   getBoardresponse = json.loads(getBoardresponse)
```

Comment:
This is probably a little clearer done in one line - otherwise getBoardResponse is storing two very different data structures which we should try to avoid
```suggestion
   getBoardresponse = json.loads(response.text)
```